Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -841,3 +824,41 @@ extension ManifestPathOrContents: Codable {
         }
     }
 }
+
+/// Adapting Result to LLBuildValue with Codable conformance
+extension Result: LLBuildValue where Success: Codable, Failure: Codable {}
+
+extension Result: Codable where Success: Codable, Failure: Codable {
```

Comment:
See my previous comment, we can't conform stdlib's Result to Codable. I would recommend creating a wrapper struct for Result that conforms to Codable. See an example here: https://github.com/apple/swift-package-manager/blob/master/Sources/Basic/misc.swift#L117-L153